internal/support/scheduler/v2/application/scheduler/queue.go

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scheduler
+
+import (
+	"fmt"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/models"
+)
+
+func (m *manager) addInterval(interval models.Interval, sc *Executor) {
+	m.intervalToExecutorMap[interval.Name] = sc
+	m.executorQueue.Add(sc)
+}
+
+func (m *manager) deleteInterval(sc *Executor) {
```

Comment:
Why `sc` for the executor? Use something more meaningful like `e` everywhere it is used?

Reply:
Modified.